maps/level_1/map.tmx

```diff
@@ -1,16 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <map version="1.8" tiledversion="1.8.4" orientation="orthogonal" renderorder="right-down" compressionlevel="0" width="18" height="9" tilewidth="32" tileheight="32" infinite="0" nextlayerid="5" nextobjectid="23">
- <properties>
```

Comment:
I'm not sure about splitting these properties into different files...
Because maps are not generated by hand, they are by using a tool ([Tiled](https://www.mapeditor.org/)).

So it would be adding a burden to this generation to have to split it...

But maybe it's still the best option, let me just think about it a bit.

Reply:
Keep the properties part in map.tmx does not harm. We just need to copy this part to map_properties.tmx and translate map_properties.tmx.